src/Ensconce.Console/Backup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.IO;
+using System.IO.Compression;
+
+namespace Ensconce.Console
+{
+    internal static class Backup
+    {
+        internal static void DoBackup()
+        {
+            var backupSource = Arguments.BackupSource.Render();
+            var backupDestination = Arguments.BackupDestination.Render();
+
+            if (File.Exists(backupDestination) && Arguments.BackupOverwrite)
+            {
```

Comment:
Else?

Reply:
I've moved check logic here now due to the render.